lib/sync/config.go

```diff
@@ -0,0 +1,80 @@
+package sync
+
+import (
+	"time"
+
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/network"
+	"boscoin.io/sebak/lib/node"
+	"boscoin.io/sebak/lib/storage"
+	"github.com/inconshreveable/log15"
+)
+
+const (
+	SyncPoolSize             = 300
+	FetchTimeout             = 1 * time.Minute
+	RetryInterval            = 10 * time.Second
+	CheckBlockHeightInterval = 30 * time.Second
+)
+
+type Config struct {
+	storage           *storage.LevelDBBackend
+	network           network.Network
+	connectionManager network.ConnectionManager
```

Comment:
I don't think Config object should store things such as the storage, logger, or connectionmanager.
They should be used mostly to hold values.

Reply:
How about another name Builder in place of Config?